tracer/test/test-applications/integrations/Samples.AWS.SimpleNotificationService/Program.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Threading.Tasks;
+using Amazon.Runtime;
+using Amazon.SimpleNotificationService;
+using Amazon.SimpleNotificationService.Model;
+
+namespace Samples.AWS.SNS
+{
+    public class Program
+    {
+        private static async Task Main(string[] args)
+        {
+            var snsClient = GetAmazonSimpleNotificationServiceClient();
+
+            string topicName = "MyTopic";
+            string topicArn = await GetOrCreateTopicAsync(snsClient, topicName);
+            string message = "Hello, SNS!";
+
+            await PublishMessageAsync(snsClient, topicArn, message);
+        }
+
+        private static AmazonSimpleNotificationServiceClient GetAmazonSimpleNotificationServiceClient()
+        {
+            if (Environment.GetEnvironmentVariable("AWS_ACCESSKEY") is string accessKey &&
+                Environment.GetEnvironmentVariable("AWS_SECRETKEY") is string secretKey &&
+                Environment.GetEnvironmentVariable("AWS_REGION") is string region)
+            {
+                var awsCredentials = new BasicAWSCredentials(accessKey, secretKey);
+                return new AmazonSimpleNotificationServiceClient(awsCredentials, Amazon.RegionEndpoint.GetBySystemName(region));
+            }
+            else
+            {
+                var awsCredentials = new BasicAWSCredentials("x", "x");
+                var snsConfig = new AmazonSimpleNotificationServiceConfig { ServiceURL = "http://" + Host() };
+                return new AmazonSimpleNotificationServiceClient(awsCredentials, snsConfig);
+            }
+        }
+
+        private static string Host()
+        {
+            return Environment.GetEnvironmentVariable("AWS_SNS_HOST") ?? "localhost:9911";
```

Comment:
Two notes:
1. You should make sure to set the environment variable `AWS_SNS_HOST` in the `/docker-compose.yml` similar to `AWS_SQS_HOST`.
2. Where is port 9911 used? Should this be updated to 4566 to match the `localstack` docker compose service?

Reply:
not sure why 9911 port is left in there, nice catch